java-security/src/test/java/com/sap/cloud/security/servlet/XsuaaTokenAuthenticatorTest.java

```diff
@@ -138,6 +148,67 @@ public void validateRequest_invalidToken_listenerIsCalled() {
 		Mockito.verifyNoMoreInteractions(validationListener2);
 	}
 
+    @Test
```

Comment:
maybe you like to add another test case with ias token and withEnvironmentVariable("IAS_XSUAA_XCHANGE_ENABLED", "false")

Reply:
good point 👍 thanks